Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -73,7 +73,35 @@ open class _ChatChannelListItemView<ExtraData: ExtraDataTypes>: _ChatChannelSwip
         .itemSubviews
         .unreadCountView.init()
         .withoutAutoresizingMaskConstraints
-    
+
+    open var titleText: String? {
+        if let channel = content.channel {
+            return uiConfig.channelList.channelNamer(channel, content.currentUserId)
+        } else {
+            return nil
+        }
+    }
+
+    /// The `subtitleLabel` will show the current typing member or the last message in the channel.
+    open var subtitleText: String? {
```

Comment:
`subtitleLabel` -> `subtitleText` or was it intentional?

Reply:
Typo, fixed.